wasm/generateInterface.js

```diff
@@ -348,15 +349,23 @@ function generateManifest (interfaceManifest) {
   return json
 }
 
-const json = generateManifest(interfaceManifest)
+// generateManifest mutates the interface input, so use a copy
+const interfaceManifestCopy = JSON.parse(JSON.stringify(interfaceManifest))
+
+let syncIfaceJson = generateManifest(interfaceManifest, {'useAsyncAPI': false})
+let asyncIfaceJson = generateManifest(interfaceManifestCopy, {'useAsyncAPI': true})
```

Comment:
Is `generateManifest` changing the first parameter so you need a copy?

Also how about naming it `asyncInterfaceJson` or `asyncJson`?